dbt_copilot_helper/templates/svc/maintenance_pages/migration.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html lang="en-GB">
+<head>
+    <title>We'll be back soon... | DBT</title>
+    <link rel="stylesheet"
+          href="https://sso.trade.gov.uk/static/stylesheets/sso.css">
+</head>
+<body class="govuk-template__body">
+<header class="govuk-header " role="banner" data-module="govuk-header">
+    <div class="govuk-header__container govuk-width-container">
+        <div class="govuk-header__logo"></div>
+    </div>
+</header>
+<div class="govuk-width-container">
+    <main class="govuk-main-wrapper" id="main-content" role="main">
+        <h1 class="govuk-heading-l">Scheduled Maintenance</h1>
+        <p class="govuk-body-l">
+            The service is currently unavailable. Don't worry, we'll be back by the end of the day.
```

Comment:
Would be nice if the body message could be configurable in some way.

Reply:
Yeah that would be nice, but this and the default page cover our current use cases.